### DISCUSSION TITLE
[gatsby-plugin-image] childImageCharp is empty with .svg files

### DISCUSSION THREAD

**Discussion by maweo-mathis** [gatsbyjs/gatsby#30886] [Help]
Helllo,

I use the new gatsby-plugin-image. I want to use that plugin to display a .svg file served by wordpress (ACF).
But the childImageCharp Object is empty.

```
icon {
  localFile {
    childImageSharp {
      gatsbyImageData(layout: FULL_WIDTH)
    }
  }
}
```

`console.log("icon", socialMediaRepeater[0].icon)`

![image](https://user-images.githubusercontent.com/78079355/114855123-dfa2eb80-9de5-11eb-9f20-cae33bbf0db5.png)

If I change the file to a .png File, everything works fine

**Comment by LekoArts** [comment]
This is working as expected as `SVG` is not supported as an input & output file. You'll need to display the svg with the normal `<img />` tag

**Reply by maweo-mathis** [reply]
okay thanks for your reply, but still the childImageSharp is null so I need another fragment or don't need to use getImage()?

**Reply by LekoArts** [reply]
Check if you get a `publicURL` somewhere:

```graphql
icon {
  localFile {
    publicURL
  }
}
```

**Reply by maweo-mathis** [reply]
yes that works, but I need the SVG to display as inline `<svg>` ... is that possible?

**Reply by LekoArts** [reply]
You'll need to adapt the code from https://github.com/axe312ger/gatsby-transformer-inline-svg as it's specific to Contentful. But yes, it's possible
